src/start.sh

```diff
@@ -70,18 +77,11 @@ start() {
     # We need the PID of the capsh process so that we can wait for it to finish
     CAPSH_PID=$!
 
-    # Get the FTL log file path from the config
-    FTLlogFile=$(getFTLConfigValue files.log.ftl)
-
-    # Wait until the log file exists before continuing
-    while [ ! -f "${FTLlogFile}" ]; do
-        sleep 0.5
-    done
-
-    #  Wait until the FTL log contains the "FTL started" message before continuing
-    while ! grep -q '########## FTL started' "${FTLlogFile}"; do
-        sleep 0.5
-    done
+    # Wait for FTL to start by monitoring the FTL log file for the "FTL started" line
+    if ! timeout 30 tail -F -c +$((startFrom + 1)) -- "${FTLlogFile}" | grep -q '########## FTL started'; then
+        echo "  [!] ERROR: FTL failed to start within 30 seconds, stopping container"
+        exit 1
+    fi
```

Comment:
Would it be helpful to include something in the error message that says "Did not find 'FTL started' message in ${FTLlogFile} in 30 seconds" instead of just echoing that FTL failed to start? At least give users the information of how we determined that FTL has not started.